cmd/operator/main.go

```diff
@@ -42,8 +42,11 @@ func main() {
 		utils.Log.Info().Msg("NetworkPolicies generation is disabled.")
 	}
 	services.WatchProjects()
-
-	timerKubiRefresh := time.NewTicker(10 * time.Minute)
+        
+	//os.Setenv("timer_refresh" , "10")
+	timerKubiRefresh := time.NewTicker(5 * time.Minute)
```

Comment:
I don't think it's the right idea to change the default value in the implementation of an e2e test.

It makes more sense to me to add a variable, which defaults to current value of 10 minutes, and override it for the test.

Reply:
done !